mlflow/store/db_migrations/versions/0a8213491aaa_drop_duplicate_killed_constraint.py

```diff
@@ -31,7 +31,8 @@ def upgrade():
     # operation is expected to fail under certain circumstances, we execute `drop_constraint()`
     # outside of the batch operation context.
     try:
-        op.drop_constraint(constraint_name="status", table_name="runs", type_="check")
+        # op.drop_constraint(constraint_name="status", table_name="runs", type_="check")
```

Comment:
Is it safe to turn this into a no-op?

Reply:
I found attempting a non-existing constraint results in an error:

https://github.com/harupy/mlflow/runs/4763865255?check_suite_focus=true#step:9:1235

I just tried commenting out this line to see if this line throws the error.

I think we need to drop the `status` constraint if it exists.

Reply:
I'll reopen this PR on `mlflow/mlflow`.